recipes/fastgltf/all/conanfile.py

```diff
@@ -113,3 +111,5 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = ["fastgltf"]
+        if "20" in str(self.settings.get_safe("compiler.cppstd")):
```

Comment:
Shouldn't this be `self.settings.compiler.get_safe("cppstd")` as above in line 95?

Reply:
It's safer on this way in case there is no `compiler` listed in the profile.